client/asset/eth/eth.go

```diff
@@ -273,6 +277,15 @@ func (eth *ExchangeWallet) OwnsAddress(address string) (bool, error) {
 //
 // TODO: Return Immature and Locked values.
 func (eth *ExchangeWallet) Balance() (*asset.Balance, error) {
+	eth.lockedFundsMtx.Lock()
+	eth.lockedFundsMtx.Unlock()
+
+	return eth.balanceImpl()
+}
+
+// balanceImpl returns the total available funds in the account.
+// This function expects eth.lockedFundsMtx to be held.
+func (eth *ExchangeWallet) balanceImpl() (*asset.Balance, error) {
```

Comment:
What is `Impl`?

Reply:
I would guess implementation.

Reply:
Yeah implementation.

Reply:
I guess I don't understand why that's in the function name. Why not just `balance`? What is "balance implementation"?